src/components/input/bl-input.test.ts

```diff
@@ -44,6 +45,21 @@ describe("bl-input", () => {
     expect(el.shadowRoot?.querySelector("input")?.getAttribute("type")).to.equal("number");
   });
 
+  it('should call showPicker if "showPicker" is in HTMLInputElement.prototype', async () => {
+    const el = await fixture<BlInput>(html`<bl-input type="input"></bl-input>`);
+    const spy = stub(el.validationTarget, "showPicker");
+
+    el.showPicker();
+
+    el.requestUpdate();
+    await el.updateComplete;
+
+    expect(HTMLInputElement.prototype).to.have.ownProperty("showPicker");
```

Comment:
Do we need to check this? It's related to the browser support and we have nothing to do if this test fails.